.github/workflows/test.yml

```diff
@@ -67,16 +67,50 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install requirements
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install --requirement requirements.txt
-    - name: UnitTest with pytest
+        pip install --upgrade pip
+        pip install --requirement requirements.txt
+    - name: UnitTest with pytest and coverage
       run: |
-        python -m pytest --random-order --random-order-bucket=global --ignore=docs --ignore=tests/test_app.py tests/
+        python -m pytest \
```

Comment:
It seems you removed `-m` option from python command line.. but it still exist here.
If you want to unify command line style, please remove `-m` option here.